hypothesis-python/tests/nocover/test_regex.py

```diff
@@ -86,6 +86,36 @@ def test_fuzz_stuff(data):
     assert regex.search(ex)
 
 
+@given(st.data())
+def test_regex_atomic_point(data):
+    pattern = "a(?>bc|b)c"
+    flags = 0
+
+    try:
+        regex = re.compile(pattern, flags=flags)
+    except (re.error, FutureWarning):
+        # Possible nested sets, e.g. "[[", trigger a FutureWarning
+        reject()
```

Comment:
Because we're using a static pattern here, this comment is wrong - and I think we'd fail a healthcheck for rejecting every single example.  Instead, let's use `@pytest.mark.skipif(sys.version_info[:2] < (3, 11), reason="new syntax)`.